src/content/peerconnection/negotiate-timing/css/main.css

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2017 The WebRTC project authors. All Rights Reserved.
```

Comment:
nit: 2021

Reply:
I cloned a 2017 sample, and then changed 90% of the lines, so I suppose 2021 is appropriate...

Reply:
Did you intend to leave this 2017? IANAL, but generally people ask me to update the copyright when I make a copy.